src/main/java/net/dv8tion/jda/api/audio/OpusPacket.java

```diff
@@ -48,38 +57,93 @@ public OpusPacket(@Nonnull AudioPacket packet, long userId, @Nullable Decoder de
         this.opusAudio = packet.getEncodedAudio().array();
     }
 
+    /**
+     * The sequence number of this packet. This is used as ordering key for {@link #compareTo(OpusPacket)}.
+     * <br>A char represents an unsigned short value in this case.
+     *
+     * <p>Note that packet sequence is important for decoding. If a packet is out of sequence the decode
+     * step will fail.
+     *
+     * @return The sequence number of this packet
+     *
+     * @see    <a href="http://www.rfcreader.com/#rfc3550_line548" target="_blank">RTP Header</a>
+     */
     public char getSequence()
     {
         return rawPacket.getSequence();
     }
 
+    /**
+     * The timestamp for this packet. As specified by the RTP header.
+     *
+     * @return The timestamp
+     *
+     * @see    <a href="http://www.rfcreader.com/#rfc3550_line548" target="_blank">RTP Header</a>
+     */
     public int getTimestamp()
     {
         return rawPacket.getTimestamp();
     }
 
+    /**
+     * The synchronization source identifier (SSRC) for the user that sent this audio packet.
+     *
+     * @return The SSRC
+     *
+     * @see    <a href="http://www.rfcreader.com/#rfc3550_line548" target="_blank">RTP Header</a>
+     */
     public int getSsrc()
```

Comment:
Why is this lowercase, but RawPacket's is `getSSRC`

Reply:
Because these getters were auto-generated by intellij. I didn't try to keep any kind of naming pattern. I'm open for suggestions, though.

Reply:
Lets go with `getSSRC` to match our other namings.